src/lib/seq2HLA.ml

```diff
@@ -0,0 +1,28 @@
+
+open Common
+open Run_environment
+module K = KEDSL
+
+let hla_type ?host ~work_dir ~run_with ~r1 ~r2 ~run_name =
+  let tool = Machine.get_tool run_with (`Biopamed "seq2HLA") in
+  (* Why quote this here? Seems like it easy to create a bug,
+     why not enforce this at node construction ?*)
+  let r1pt = Filename.quote r1#product#path in
+  let r2pt = Filename.quote r2#product#path in
```

Comment:
@hammer more tests would be awesome indeed `#HelpWanted`

(testing installation + run in normal use cases already takes lots of hours to people; I understand that trying corner cases and reading GATK/samtools corresponding error messages is often left out... espcially since some changes in, say, `bwa` can make semi-broken bams for BQSR much later in user pipelines)